x-pack/plugins/watcher/public/application/license_prompt.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiEmptyPrompt, EuiLink, EuiPageContent_Deprecated as EuiPageContent } from '@elastic/eui';
+import { LicenseManagementLocator } from '@kbn/license-management-plugin/public/locator';
+
+export const LicensePrompt = ({
+  message,
+  licenseManagementLocator,
+}: {
+  message: string | undefined;
+  licenseManagementLocator?: LicenseManagementLocator;
+}) => {
+  const licenseManagementUrl = licenseManagementLocator?.useUrl({ page: 'dashboard' });
+  // if there is no licenseManagementUrl, the license management plugin might be disabled
+  const promptAction = licenseManagementUrl ? (
+    <EuiLink href={licenseManagementUrl}>
```

Comment:
UX nit: It seems that when we follow this link we do a full page reload, is there a way to avoid that? 🤔

Reply:
Great catch, @sabarasaba! I'll open an issue for that, I think the [recommended](https://docs.elastic.dev/kibana-dev-docs/routing-and-navigation#navigating-between-kibana-apps) way is to wrap the app in `RedirectAppLinks`. That way all links in the app will work as a SPA navigation without the page reload.